autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/saving/declines/AutofillDisablingDeclineCounter.kt

```diff
@@ -136,6 +136,6 @@ class AutofillDisablingDeclineCounter @Inject constructor(
     }
 
     companion object {
-        private const val GLOBAL_DECLINE_COUNT_THRESHOLD = 3
+        private const val GLOBAL_DECLINE_COUNT_THRESHOLD = 2
```

Comment:
Found this unrelated to the PR, just in case it's not expected to be here.

Reply:
Caused because another branch in the stack was already merged. I've restacked so the change disappears from the PR. Thanks for flagging 👍